test/lib/webmention/crawl_test.rb

```diff
@@ -1,4 +1,4 @@
-require_relative '../../test_helper'
+require 'test_helper'
```

Comment:
In these test files, changing this to `require 'test_helper'` fixed some oddness in coverage reporting.